src/esw/brushless.py

```diff
@@ -237,6 +247,29 @@ async def update(self) -> None:
         ):
             await self._connect()
 
+    @staticmethod
```

Comment:
can u add comment that this is just a copy of moteus library and why we have it

Reply:
Yes def good idea